composables/useChain.ts

```diff
@@ -31,6 +35,7 @@ export default function () {
   })
 
   const availableChains = computed(() => availablePrefixes())
+  const availableChainsWithIcon = computed(() => availablePrefixWithIcon())
```

Comment:
`computed(availablePrefixWithIcon)` will work just the same

Reply:
updated